Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+name = "pypi"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+"boto3" = "*"
+onelogin = "*"
+keyring = "*"
+
+[requires]
```

Comment:
Should specify that we need python 3

Reply:
Is python version 3.5 or later?.  i think so from travis.yml file.

https://github.com/physera/onelogin-aws-cli/blob/101c30c7dd31f7e84554007c0efc022c1c81c3fe/.travis.yml#L2